GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,18 +8,144 @@
+                return cell
+            }
+        case 2:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea1", for: indexPath) as? Sea1TableViewCell{
+                cell.nameLabel1.text = GOTEpisode.num3[indexPath.row].name
+                
+                cell.epNumber.text = "Episode: \(GOTEpisode.num3[indexPath.row].number) Season: \(GOTEpisode.num3[indexPath.row].season)"
+                
+                cell.image1.image = UIImage(named:  GOTEpisode.num3[indexPath.row].originalImageID )
+                return cell
+            }
+        case 3:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea2", for: indexPath) as? Sea2TableViewCell{
+                cell.nameLabel2.text = GOTEpisode.num4[indexPath.row].name
+                
+                cell.epLabel2.text = "Episode: \(GOTEpisode.num4[indexPath.row].number) Season: \(GOTEpisode.num4[indexPath.row].season)"
+                
+                cell.image2.image = UIImage(named:  GOTEpisode.num4[indexPath.row].originalImageID )
+                return cell
+            }
+        case 4:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea1", for: indexPath) as? Sea1TableViewCell{
+                cell.nameLabel1.text = GOTEpisode.num5[indexPath.row].name
+                
+                cell.epNumber.text = "Episode: \(GOTEpisode.num5[indexPath.row].number) Season: \(GOTEpisode.num5[indexPath.row].season)"
+                
+                cell.image1.image = UIImage(named:  GOTEpisode.num5[indexPath.row].originalImageID )
+                return cell
+            }
+        case 5:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea2", for: indexPath) as? Sea2TableViewCell{
+                cell.nameLabel2.text = GOTEpisode.num6[indexPath.row].name
+                
+                cell.epLabel2.text = "Episode: \(GOTEpisode.num6[indexPath.row].number) Season: \(GOTEpisode.num6[indexPath.row].season)"
+                
+                cell.image2.image = UIImage(named:  GOTEpisode.num6[indexPath.row].originalImageID )
+                return cell
+            }
+        case 6:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea1", for: indexPath) as? Sea1TableViewCell{
+                cell.nameLabel1.text = GOTEpisode.num7[indexPath.row].name
+                
+                cell.epNumber.text = "Episode: \(GOTEpisode.num7[indexPath.row].number) Season: \(GOTEpisode.num7[indexPath.row].season)"
+                
+                cell.image1.image = UIImage(named:  GOTEpisode.num7[indexPath.row].originalImageID )
+                return cell
+            }
+            
+        default:
+            print("nice try")
+        }
+        return UITableViewCell()
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
```

Comment:
Same here: You're hard coding the number of sections but that number is just an Int, does not correspond the number of seasons you have in the case that changes.
The right way to return data is based on what you have on your model:
e.g
    return arrOfSeasons.count